src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -7199,4 +7199,211 @@ static void Main()
         Assert.Equal("(7,9)", locationSpecifier.GetDisplayLocation());
         AssertEx.Equal("""[global::System.Runtime.CompilerServices.InterceptsLocationAttribute(1, "jB4qgCy292LkEGCwmD+R6FIAAAA=")]""", locationSpecifier.GetInterceptsLocationAttributeSyntax());
     }
+
+    [Fact]
+    public void ConditionalAccess_01()
+    {
+        var source = CSharpTestSource.Parse("""
+            class C
+            {
+                void M() => throw null!;
+
+                static void Main()
+                {
+                    var c = new C();
+                    c?.M();
```

Comment:
It would be good for tests on new interceptable locations to also test `GetInterceptorMethod` API